modules/installation-vsphere-machines.adoc

```diff
@@ -118,15 +133,18 @@ $ export IPCFG="ip=192.168.100.101::192.168.100.254:255.255.255.0:::none nameser
 $ govc vm.change -vm "<vm_name>" -e "guestinfo.afterburn.initrd.network-kargs=${IPCFG}"
 ----
 +
+... Set the extra config properties of the control plane, compute, and bootstrap Ignition config files before booting a VM from an OVA in vSphere. For example, to change the bootstrap Ignition config:
++
+[source,terminal]
+----
+$ govc vm.change -vm "<vm_name>" -e "guestinfo.bootstrap.ign=${IPCFG}"
+----
++
```

Comment:
From Slack comment: 

> 5g where it describes afterburn with govc commands. We could do that for ignition config as well

I'm not sure if this is suggesting to place this in the first steps where the user is locating and converting Ignition configs, or alongside this step (5g) with afterburn. I've gone with the latter, though I don't understand what the IPCFG variable is doing and if it should apply. 

@jcpowermac Could you specify where you think this step belongs and what command the user should run? Is it also optional?

Reply:
@bobfuru this is kinda what I was thinking:
```
govc vm.change -vm "<vm_name>" -e "guestinfo.ignition.config.data.encoding=base64"
govc vm.change -vm "<vm_name>" -e "guestinfo.ignition.config.data=$(cat bootstrap.ign)"
```
Would need to be done to all 7 virtual machines though.

Reply:
IIUC, these steps are better suited for the beginning of the procedure. Basically, they should find the bootstrap, master, and worker Ignition configs, convert to Base64, and store in the ExtraConfig of the VM using your command examples. 

Assuming that to be the case, I relocated these to the top (steps 1-3).